paimon-flink/paimon-flink-common/pom.xml

```diff
@@ -91,6 +93,26 @@ under the License.
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-connector-mongodb-cdc</artifactId>
+            <version>2.4.1</version>
```

Comment:
Does this need to be consistent with flink cdc mysql?

Reply:
MongoDB CDC introduced the scan.startup.mode parameter as of version 2.4, offering the choices of "initial," "latest-offset," and "timestamp." This feature holds significant importance for data synchronization. This is precisely why I opted not to align versions with MySQL CDC.

reference link:
2.3:https://ververica.github.io/flink-cdc-connectors/release-2.3/content/connectors/mongodb-cdc.html
2.4:https://ververica.github.io/flink-cdc-connectors/release-2.4/content/connectors/mongodb-cdc%28ZH%29.html